api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/AggregatedStatsQueryRequest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2014-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.metrics.api.jaxrs;
+
+import java.util.List;
+import java.util.Objects;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * @author jtakvorian
+ */
+public class AggregatedStatsQueryRequest {
```

Comment:
The same class may be used for avail to request mixed availability.
So maybe it should be renamed "MixedStatsQueryRequest"